Tests/SwiftRefactorTest/ConvertComputedPropertyToStored.swift

```diff
@@ -0,0 +1,67 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftRefactor
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import XCTest
+import _SwiftSyntaxTestSupport
+
+final class ConvertComputedPropertyToStoredTest: XCTestCase {
+  
+  func testToStored() throws {
+    let baseline: DeclSyntax = """
+      var defaultColor: Color { Color() }
+      """
+    
+    let expected: DeclSyntax = """
+      let defaultColor: Color = Color()
+      """
+    
+    try assertRefactorConvert(baseline, expected: expected)
+  }
+  
+  func testToStored2() throws {
```

Comment:
How about giving this a little more descriptive name, eg. `testToStoredWithMultipleStatementsInAccessor`